vuepress/docs/docs/tools.md

```diff
@@ -3,6 +3,7 @@
 Caliban comes with a module called `caliban-tools` that exposes some useful features:
 - all the code generation features from `caliban-codegen-sbt`, so that you can use them without sbt: see `caliban.tools.Codegen`.
 - a client for GraphQL introspection: see `caliban.tools.IntrospectionClient`.
+- utilities for [stitching GraphQL schemas](/stitching.html).
```

Comment:
```suggestion
- utilities for [stitching GraphQL schemas](stitching.md).
```

I think it should be this. You can double check by running `yarn docs:dev` in the vuepress folder.

Reply:
🙏 

Fixed in 45933b8.